### PR TITLE
Fix okd-scos image tags

### PR DESCRIPTION
**Pull request by andfasano**
Use the correct scos image tags

### CODE REVIEW

**Comment by andfasano** [comment]
cc @LorbusChris

File: ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-main__okd-scos.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ images:
   dockerfile_path: Dockerfile
   from: base
   inputs:
-    origin_4.15_installer:
+     okd_scos_4.15_installer:
```
Comment (LorbusChris) [inline ci-operator/config/openshift/machine-os-images/openshift-machine-os-images-main__okd-scos.yaml:19]:
```suggestion
    okd_scos_4.15_installer:
```

**Review by LorbusChris — CHANGES_REQUESTED** [review]

**Pull request merged**